### PR TITLE
Style

### PR DESCRIPTION
**Pull request by jasonfelice**
### Style Bookstore

- Copy the template style from Zeplin.
  - [x] Styled each component as given in the template.

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
+$whiteTwo: #e8e8e8;
+$background: #fafafa;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```
Comment (Ismailco) [inline src/App.scss:10]: - [ ] The page should look the same as the Zeplin design, please add everything in the design e.g. `progress circle`, `profile button in the nav bar`, `chapter`...

|Zeplin design|Your design|
|-|-|
|Screenshot_20220609_052237|Screenshot_20220609_051908|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @jasonfelice,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @jasonfelice,

You've done well addressing concerns from the previous review 👏🏼.
Tasks for this project are now successfully implemented 💯.

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jasonfelice** [comment]
> Hi, @jasonfelice,
> 
> You've done well addressing concerns from the previous review 👏🏼. Tasks for this project are now successfully implemented 💯.
> 
> ## Highlights
> * Professional README ✔️
> * Clean code ✔️
> * Good commit messages ✔️
> 
> All the best with the next project.
> 
> You are free to merge this branch :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. **Please, remember to tag me in your question so I can receive the notification.**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks

**Pull request merged**